SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -214,6 +216,29 @@ void SaveSchemaDatabase()
 	});
 }
 
+void LoadDefaultGameMode()
+{
+	// Get the default GameMode from the DefaultEngine.ini
+	FString DefaultGameModePath;
+	GConfig->GetString(
+		TEXT("/Script/EngineSettings.GameMapsSettings"),
+		TEXT("GlobalDefaultGameMode"),
+		DefaultGameModePath,
+		GEngineIni
+	);
+
+	if (DefaultGameModePath.IsEmpty())
+	{
+		return;
+	}
+
+	// Load the default GameMode so it is ready for schema generation.
+	if (StaticLoadClass(AGameModeBase::StaticClass(), NULL, *DefaultGameModePath) == nullptr)
```

Comment:
This will get loaded and then potentially immediately freed on the next GC cycle. As the schema generator runs on a separate thread, that might happen before it gets to it. We'll want to hold onto the object until the schema generator finished.